Makefile

```diff
@@ -142,30 +142,24 @@ upgrade-package: ## update just one package to the latest usable release
 check-types: ## run static type-checking tests
 	mypy
 
-docker_build:
```

Comment:
Is it possible to keep the separation between build and push steps?  I can envision the need to iterate on the build process locally for development/debugging, without wanting to push the results up to Docker Hub.

Reply:
working on it.